### ISSUE TITLE
problem to send mesage without segment separator in the end of sections

### ISSUE THREAD

**Issue by franciscoolmfdz** [senaranya/HL7#13]
Expected message:
MSH|....|....|||2.5
PID|1|....
Received Message:
MSH|....|....|||2.5|
PID|2|....|

the last field separator in the end of segment is the problem.
probably , can you add an option on send? like this?
$hl7Msg = str_replace(["|\r", "|\n"], ["\r", "\n"], $hl7Msg);

**Comment by senaranya** [comment]
Thank you for the change request. 

Changing current behavior of `send()` method may break existing usage. I can add a config option `SEGMENT_ENDING_BAR` to Message class. So you can create your message using:
```php
$msg = new Message(file_get_contents($hl7File), ['SEGMENT_ENDING_BAR' => false]);
```

This will remove the ending `|` while sending the HL7.

Will that help?

**Comment by franciscoolmfdz** [comment]
yeah,thanks for your reply.

**Comment by senaranya** [comment]
Fixed @ release 1.5.0
